sdk/core/azure-core/azure/core/trace/base.py

```diff
@@ -0,0 +1,83 @@
+import functools
+from os import environ
+import re
+
+from azure.core.trace.context import tracing_context
+from azure.core.trace.abstract_span import AbstractSpan
+from azure.core.trace.span import SUPPORTED_TRACE_IMPLEMENTATIONS, OpenCensusSpan
+
+
+def use_distributed_traces(func):
+    # type: (Callable[[Any], Any]) -> Callable[[Any], Any]
+    @functools.wraps(func)
+    def wrapper_use_tracer(self, *args, **kwargs):
+        # type: (Any) -> Any
+
+        name = (
+            self.__class__.__name__
+            + "."
+            + func.__name__
+            + "("
+            + ", ".join([str(i) for i in args])
```

Comment:
IMO we may not want to include actual argument values here, as they could potentially contain secret keys, PII, etc. Would be good to get other opinions, though.

Reply:
that is a good point. I agree with you.